content/en/docs/reference/glossary/quantity.md

```diff
@@ -10,22 +10,23 @@ aka:
 tags:
 - core-object
 ---
- A whole-number representation of small or large numbers using SI suffixes.
+ A whole-number representation of small or large numbers using [SI](https://en.wikipedia.org/wiki/International_System_of_Units) suffixes.
 
 <!--more-->
 
 Quantities are representations of small or large numbers using a compact,
-whole-number notation with SI suffixes.  Fractional numbers are represented
+whole-number notation with [SI](https://en.wikipedia.org/wiki/International_System_of_Units) suffixes.  Fractional numbers are represented
```

Comment:
I'm afraid you only need to link for the first occurence of a term, not every occurrence.

Reply:
(☞ﾟ∀ﾟ)☞  Done!  And I fixed the commit history.